request_a_govuk_domain/request/templates/registrant_details.html

```diff
@@ -4,7 +4,13 @@
 {% block title %}Registrant details{% endblock %}
 
 {% block beforeContent %}
-  {% url "domain_confirmation" as back_url %}
+  {% if route.primary == 1 or route.primary == 3 %}
+    {% url "domain_confirmation" as back_url %}
+  {% elif route.primary == 2 and route.secondary == 5 and route.tertiary == 8 %}
+    {% url "minister" as back_url %}
+  {% else %}
+    {% url "minister_upload_confitm" as back_url %}
```

Comment:
Typo here. Has this been tested?

Reply:
Well spotted. I also added tests that would have caught it.